geoutils/georaster.py

```diff
@@ -124,7 +124,7 @@ def __init__(self, filename, bands=None, load_data=True, downsampl=1,
             self.is_loaded = True
             if isinstance(filename, str):
                 self._is_modified = False
-                self._disk_hash = hash((hashlib.sha1(self._data), self.transform, self.crs, self.nodata))
+                self._disk_hash = hash((self._data.tobytes(), self.transform, self.crs, self.nodata))
```

Comment:
Is this triggered when creating from another Raster instance as well?

Reply:
No, this only works with input filename. For the rest, the `is_modified` will stay as `None`.

Reply:
It's my afternoon brain thinking. Of course it shouldn't be triggered without a file on the disk!